libraries/chain/finality/finalizer.cpp

```diff
@@ -267,7 +300,7 @@ void my_finalizers_t::load_finalizer_safety_info_v1(fsi_map& res) {
       bls_public_key pubkey;
       my_finalizers_t::fsi_t fsi;
       fc::raw::unpack(persist_file, pubkey);
-      fc::raw::unpack(persist_file, fsi);
+      unpack_v1(persist_file, fsi);
       res.emplace(pubkey, fsi);
```

Comment:
Why these safey.dat file changes? Just clean up or is there an actual change here?

Reply:
Not a cleanup. We cannot use the `fc::raw::unpack(persist_file, fsi);` because the `fsi` includes `last_vote` and `lock`, which are of type `block_ref`, and `block_ref` now includes two new members (the active and pending generations)